stripe/src/main/java/com/stripe/android/model/Token.java

```diff
@@ -1,62 +1,91 @@
 package com.stripe.android.model;
 
+import android.support.annotation.NonNull;
+import android.support.annotation.StringDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 import java.util.Date;
 
 /**
  * The model of a Stripe card token.
  */
 public class Token extends com.stripe.model.StripeObject {
-    private final String id;
-    private final Date created;
-    private final boolean livemode;
-    private final boolean used;
-    private final Card card;
+
+    @Retention(RetentionPolicy.SOURCE)
+    @StringDef({TYPE_CARD})
+    public @interface TokenType {}
+    public static final String TYPE_CARD = "card";
```

Comment:
This is overkill considering that the only type of token that android currently handles is "card", but eventually we're going to include bank account tokens and potentially PII tokens.